tests/DateTimeTest.php

```diff
@@ -146,6 +146,22 @@ public function provider_test_change_cascade()
 		];
 	}
 
+	public function test_change_on_timezone_option()
+	{
+		$expectedTimezone = 'Asia/Taipei';
+		$now = new DateTime();
+		$options = [
+
+			'timezone' => $expectedTimezone,
+
+		];
+		$result = $now->change($options);
+		$resultTimezone = $result->getTimezone();
+
+		$this->assertInstanceOf(\DateTimezone::class, $resultTimezone);
```

Comment:
We don't really need this assertion, `getTimezone()` only returns `DateTimezone`.

https://www.php.net/manual/en/datetime.gettimezone.php

Reply:
Yes. This assertion is not necessary.